app/validation/validator.py

```diff
@@ -84,6 +83,9 @@ def validate_questionnaire(self, json_to_validate):
             answers_with_parent_ids = self._get_answers_with_parent_ids(
                 json_to_validate
             )
+            option_value_to_answer_id_map = self._get_option_value_to_answer_id_map(
```

Comment:
As this is a map of answer ids to option values I think it should be named `answer_id_to_option_values_map`.